doc/install.text

```diff
@@ -0,0 +1,124 @@
+The following notes should allow you to install the LSST stack and the PFS DRP.
+After the installation procedure example commands are given to show you how to
+use the pipeline. The commands have been tested on an Arch Linux machine as well
+as on Mac OS X.
+
+* Install lsst binary distribution (https://pipelines.lsst.io/install/conda.html)
+* Install git-lfs (https://git-lfs.github.com/)
+* Install matplotlib:
+
+conda install matplotlib
+
+
+If you have a very fresh compiler version, this could lead to incompatibilities 
+with the precompiled LSST stack. In case you run into trouble compiling or testing
+the LSST stack please install your compiler in anaconda:
+
+conda install gcc
+
+
+* Create a directory for the PFS-DRP repositories:
+
+mkdir ~/stella-git
+cd ~/stella-git
+
+* Clone the relevant git repositories:
+
+git clone https://github.com/Subaru-PFS/drp_stella.git
+git clone https://github.com/Subaru-PFS/drp_stella_data.git
+git clone https://github.com/Subaru-PFS/obs_pfs.git
+
+
+* Install a newer version of eigen:
+
+eups distrib install eigen 3.2.5.lsst2
+
+
+* Declare the git repositories in Linux (replace <Linux64> with <DarwinX86> for Mac OS X):
+
+eups declare obs_pfs 1.0 -c -f Linux64 -r ~/stella-git/obs_pfs
+eups declare drp_stella_data 1.0 -c -f Linux64 -r ~/stella-git/drp_stella_data
+eups declare drp_stella 1.0 -c -f Linux64 -r ~/stella-git/drp_stella
+
+
+* Setup the pipeline. As obs_pfs and drp_stella_data are required packages they are set up automatically:
+
+setup drp_stella 1.0 -v
+
+During this step the environment variables $DRP_STELLA_DIR, $DRP_STELLA_DATA_DIR, and $OBS_PFS_DIR are automatically 
+created as well:
+
+echo $DRP_STELLA_DIR
+
+should show you:
+
+~/stella-git/drp_stella
+
+
+* Build the pipeline:
+
+cd drp_stella
+scons opt=3 -j8
+
+
+* Create the defects database (done during compilation of the code):
+
+cd ../obs_pfs
+scons opt=3 -j8
+
+
+* Now for using the pipeline. Raw test data are in $DRP_STELLA_DATA_DIR/tests/data/raw/ (3 Biases, 3 Darks, 1 Flat, 1 Arc). 
+First we need to create a directory (actually 2) where we want to store pipeline outputs. 
+Let's assume you have a directory ~/spectra and do everything related to spectra there:
+
+mkdir ~/spectra/PFS
+mkdir ~/spectra/PFS/CALIB
```

Comment:
You can do this with just one command: `mkdir -p ~/spectra/PFS/CALIB`. That also means we can cut the quibbling about whether we're making one directory or two.

Rather than assuming we're working in the home directory, might be nice to be less explicit. But that's not a big deal.